modules/base/main/index.scss

```diff
@@ -184,4 +179,13 @@ input[type=radio] {
 
 hr {
   border: 1px solid transparent;
+}
+
+input,
+textarea,
+select {
+  &:disabled {
+    cursor: not-allowed;
+    opacity: 0.5;
+  }
```

Comment:
Moved this to the bottom so that the styles aren't overriden for checkbox/radio inputs